platform/android/java/lib/build.gradle

```diff
@@ -105,7 +105,7 @@ android {
         def taskName = getSconsTaskName(buildType)
         tasks.create(name: taskName, type: Exec) {
             executable sconsExecutableFile.absolutePath
-            args "--directory=${pathToRootDir}", "platform=android", "target=${releaseTarget}", "android_arch=${defaultAbi}", "-j" + Runtime.runtime.availableProcessors()
+            args "--directory=${pathToRootDir}", "platform=android", "target=${releaseTarget}", "arch=${defaultAbi}", "-j" + Runtime.runtime.availableProcessors()
```

Comment:
You'll also need to update the [`supportedAbis`](https://github.com/godotengine/godot/blob/15aea89868d16d0310a229d831e4a347a38dc1c3/platform/android/java/build.gradle#L24) values from which `defaultAbi` is selected.